pkg/addons/addons.go

```diff
@@ -570,3 +572,59 @@ func UpdateConfig(cc *config.ClusterConfig, enabled []string) {
 		}
 	}
 }
+
+// CheckPaused checks whether the cluster is paused before enable/disable an addon.
+func CheckPaused(profile string, val string) error {
```

Comment:
Any reason `val` can't be a bool?

Reply:
I was thinking to keep consistent with `EnableOrDisableAddon`, which also makes `val` as a string. Now switch to use bool.